hicommonwealth/commonwealth
# Directory Page & Related Communities

## Link to Issue
Closes: #4679 

## Description of Changes
- Added "directory page section" in admin panel on "/manage" community page
- Added "Directory" item in left sidebar that is displayed depending on the toggle in admin "directory page section"
- Added "/directory" page that shows list of related communities based on chain selected in admin "directory page section"
- Added search, table view and grid view

## Test Plan
- as community admin go to manage community page and scroll to "directory page section"
- toggle switch to make "/directory" page available (only after hitting save button)
- default chain should be already selected, but admin can pick different chain from the dropdown
- click "go to directory" button to visit "/directory" page
- page should list all communities that are based on selected chain
- use search to narrow the results (search works by community name)
- toggle view between table/grid

Available on https://commonwealth-demo.herokuapp.com/

https://github.com/hicommonwealth/commonwealth/assets/14819225/b92373e3-cdd7-4f63-ae42-4dcc6fe3112c




## Deployment Plan

1. no special requests
